src/execution_plan/ops/op_conditional_traverse.c

```diff
@@ -57,16 +55,15 @@ OpResult CondTraverseConsume(OpBase *opBase, QueryGraph* graph) {
     }
 
     /* Get node from current column. */
```

Comment:
This comment should say "row".

Reply:
We're picking nodes along the current column. whenever the column has NNZ in it.

Reply:
Ah, I see - you're right, the original makes sense!